<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>522</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../view_img/fondo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:00715909, stop:0 rgba(0, 0, 0, 35), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));

border: 3px solid white; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>451</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>411</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,35);
border-radius:13px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>218</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Restaurar la&lt;/p&gt;&lt;p&gt;Base de Datos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>68</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../view_img/restaurar.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>-10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>49</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/IMG/metrologo.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../view_img/metrologo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>36</y>
       <width>251</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:00715909, stop:0 rgba(0, 0, 0, 20), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btRestaurar">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>569</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(255, 170, 0);
}

QPushButton:pressed {
	background-color: rgb(217, 145, 0);
}
</string>
     </property>
     <property name="text">
      <string>Restaurar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btRegresar">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>569</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(255, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(166, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Regresar</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>522</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="GB_2_CLIENT">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>317</y>
       <width>311</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: black;
}

QGroupBox{
color: white
}
</string>
     </property>
     <property name="title">
      <string>Selecciona el BackUp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QListView" name="lst_vw">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>34</y>
        <width>261</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListView {
    background-color: #f9f9f9;
    color: #333;
    border: 1px solid #e1e1e1;
    border-radius: 6px;
    padding: 8px;
    font-size: 12px;
    font-family: Arial, sans-serif;
}

QListView::item {
    height: 28px;
    padding: 4px;
    margin: 4px 0;
    border-radius: 4px;
    background-color: #ffffff;
}

QListView::item:hover {
    background-color: #ffeb3b;
    color: #333;
}

QListView::item:selected {
    background-color: #fdd835;
    color: #ffffff;
    outline: none;
}</string>
      </property>
     </widget>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>btRestaurar</zorder>
    <zorder>btRegresar</zorder>
    <zorder>label_5</zorder>
    <zorder>warning</zorder>
    <zorder>GB_2_CLIENT</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../view_img/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
